python/lsst/ts/weatherstation/csc.py

```diff
@@ -196,7 +196,7 @@ def fault(self, code=None, report="", traceback=""):
             self.log.error("Exception trying to stop model controller.")
             self.log.exception(e)
         self.model.controller.reset_error()
-        super().fault(code=code, report=report, traceback=traceback)
+        await super().fault(code=code, report=report, traceback=traceback)
```

Comment:
A question: can stopping the model be handled in handle_summary_state? If so, you can get rid of this override of `fault`.

Reply:
Good question and, to be honest, I don't know. I prefer to keep the code as is.